styles/raindrop/catppuccin.user.css

```diff
@@ -0,0 +1,172 @@
+/* ==UserStyle==
+@name Raindrop Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/raindrop
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/raindrop
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/raindrop/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Araindrop
+@description Soothing pastel theme for Raindrop
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('app.raindrop.io') {
+  :root[data-theme="night"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="day"],
+  :root[data-theme="sunset"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  // theme selector previews
+  [class^="theme-"] {
+    &[data-index="0"],
+    &[data-index="2"] > [class^="main-"],
+    &[data-index="3"] {
+      #catppuccin(@lightFlavor, @accentColor) !important;
+    }
+    &[data-index="1"],
+    &[data-index="2"] > [class^="sidebar-"] {
+      #catppuccin(@darkFlavor, @accentColor) !important;
+    }
+  }
+
+  // mixed theme sidebar
+  [data-theme="night"] {
+    #catppuccin(@darkFlavor, @accentColor) !important;
+  }
```

Comment:
Does the theme still work if we take away the `:root` part? It's not necessary for all userstyles, and it would let us remove this "mixed theme sidebar" thing and possibly even the `&, [data-theme] {` part below.

Reply:
taking away the `:root` part breaks the light theme (the dark theme keeps working because of the "mixed theme sidebar" thing)

Reply:
however i figured out that if i just remove `:root` from the `:root` part and add `!important` to the dark theme i can remove the mixed sidebar thing
```css
  [data-theme="night"] {
    #catppuccin(@darkFlavor, @accentColor) !important;
  }
  [data-theme="day"],
  [data-theme="sunset"] {
    #catppuccin(@lightFlavor, @accentColor);
  } 
  ```